examples/multivendor/multivendor.pb.txt

```diff
@@ -51,6 +97,42 @@ nodes: {
       name: "et-0/0/3"
     }
   }
+  interfaces: {
+    key: "eth8"
+    value: {
+      name: "et-0/0/4"
+    }
+  }
+  interfaces: {
+    key: "eth9"
+    value: {
+      name: "et-0/0/5"
+    }
+  }
+  interfaces: {
+    key: "eth10"
+    value: {
+      name: "et-0/0/6"
+    }
+  }
+  interfaces: {
+    key: "eth11"
+    value: {
+      name: "et-0/0/7"
+    }
+  }
+  interfaces: {
+    key: "eth12"
+    value: {
+      name: "et-0/0/8"
+    }
+  }
+  interfaces: {
+    key: "eth13"
+    value: {
+      name: "et-0/0/9"
+    }
+  }
 }
 nodes: {
   name: "srl"
```

Comment:
xrd and nokia will need the extra service definitions including in their example textprotos

Reply:
hrm?

Reply:
nokia should look like this: https://github.com/openconfig/kne/blob/b091035fd078ad20f05ab4ea2444efad6ba36a51/examples/nokia/srlinux-services/2node-srl-ixr6-with-oc-services.pbtxt#L24

and xrd should be similar to 8000e I'd imagine: https://github.com/openconfig/kne/blob/b091035fd078ad20f05ab4ea2444efad6ba36a51/examples/cisco/8000e/8000e-ixia.pb.txt#L17

Reply:
done also found an issue where we can remove the outside setting requirement